### ISSUE TITLE
Unable to launch NES title

### ISSUE THREAD

**Issue by Pythonic-Rainbow** [FailedShack/USBHelperLauncher#5]
![image](https://user-images.githubusercontent.com/25037295/41581770-2bcc1d4c-73d2-11e8-8484-9a32ad56d6c4.png)

**Comment by Pythonic-Rainbow** [comment]
```

************** Exception Text **************
System.Net.WebException: The remote server returned an error: (404) Not Found.
   at System.Net.HttpWebRequest.GetResponse()
   at A.c07070e3f60abbdaf352210b11ada9080.c1cb398b215dd7d71f706e4325f6a66ac(String cfe300fe688845689ab9526c0dc53da7e)
   at A.cf7c96734e58d8de9864e1ea854bc2b7b.c5b82a34802a427f61b7a0d683e9456b9()
   at A.cf7c96734e58d8de9864e1ea854bc2b7b.c9a97a9b02f735137b7d889075e2595fe()
   at A.cf7c96734e58d8de9864e1ea854bc2b7b..ctor(c6114b34ba4893a53ebb4b599e0e2e3fc c804bc52f7f79bffee6fd394ef382eb76, String ce7d298d73bab0df9bcfdc208996dceec, String cbc47e314a0f1d7d5f1d07b50ba2bb8c7, Boolean ca474a42a0a5ca8e78e9445423ebb3d1f, Boolean c11e4e8c7e06a3bc593aefa84d54feebb)
   at NusHelper.Emulators.Fceux..ctor(c6114b34ba4893a53ebb4b599e0e2e3fc title, Boolean forceUpdate)
   at A.c6114b34ba4893a53ebb4b599e0e2e3fc.c04a799c99dd357cf35ae2407537c3d1c(Boolean c07532d90fa519783a3736afc4d48beb9)
   at A.cb6cc9a5b122137bc5f2718f61ab49e59.cbef1c47350bfe240624acc48226915ee()
   at A.cb6cc9a5b122137bc5f2718f61ab49e59.c37b3d3a9ef4a26c65a3fac4aa3cda6c2(c5409344e1c2b74fa8c9a4d620ccb8d48 c6fcdfcca42c1aeed647bdea8d9051572)
   at Telerik.WinControls.RadElement.OnClick(EventArgs e)
   at Telerik.WinControls.UI.RadMenuItem.OnClick(EventArgs e)
   at Telerik.WinControls.RadElement.DoClick(EventArgs e)
   at Telerik.WinControls.RadElement.RaiseBubbleEvent(RadElement sender, RoutedEventArgs args)
   at Telerik.WinControls.RadItem.RaiseBubbleEvent(RadElement sender, RoutedEventArgs args)
   at Telerik.WinControls.RadElement.RaiseRoutedEvent(RadElement sender, RoutedEventArgs args)
   at Telerik.WinControls.RadElement.RaiseBubbleEvent(RadElement sender, RoutedEventArgs args)
   at Telerik.WinControls.RadItem.RaiseBubbleEvent(RadElement sender, RoutedEventArgs args)
   at Telerik.WinControls.RadElement.RaiseRoutedEvent(RadElement sender, RoutedEventArgs args)
   at Telerik.WinControls.ComponentInputBehavior.OnMouseUp(MouseEventArgs e)
   at Telerik.WinControls.RadControl.OnMouseUp(MouseEventArgs e)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at Telerik.WinControls.RadControl.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

**Comment by Pythonic-Rainbow** [comment]
I guess the problem is the software unable to download Fceux.

**Comment by FailedShack** [comment]
Not all emulators are currently implemented in the emulator downloader, as explained in #1.
In the meantime, you may get Fceux manually and copy it to the emulators folder in the launcher directory.

**Comment by FailedShack** [comment]
Fixed via e96e90f0b173e212a7f82ea8bdc2a711ad6e949e
